great_expectations/core/usage_statistics/anonymizers/expectation_anonymizer.py

```diff
@@ -16,6 +16,9 @@ def __init__(
     def anonymize(
         self, obj: Optional["ExpectationSuite"] = None, **kwargs  # noqa: F821
     ) -> dict:
+        if obj is None:
+            raise ValueError("Cannot anonymize a null payload")
+
```

Comment:
This signature is what is required by the base class - I know this feels a bit funky but this should all be cleaned up shortly.

Reply:
Downstream logic depends on this being non-null

Reply:
But shouldn't this just use the logger? Don't we want to avoid usage stats raising any type of error so the payload fails silently?

Reply:
Or just put all logic that depends on `obj` inside this conditional?

Reply:
so we're already hitting an `AttributeError` here but yeah maybe we shouldn't raise any explicit errors